site/en/docs/android/custom-tabs/guide-interactivity/index.md

```diff
@@ -0,0 +1,197 @@
+
+//Set the pendingIntent as the action to be performed when the button is clicked.
+CustomTabsIntent intentBuilder = new CustomTabsIntent.Builder()
+    …
+    .setActionButton(shareIcon, shareDescription, pendingIntent)
+    .build();
+```
+
+## Add custom menu items
+
+Custom tabs has as many as five default actions provided by the browser: "Forward", "Page Info", "Refresh", "Find in Page" and "Open in Browser". Additionally, you can add up to five more, which will be inserted between the icon row and the browser-provided items. (See the image below.)
+
+You can access your custom actions via the three dot menu in the top right corner:
+
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/3WXYfUUMRMqzNwgIT1Ao.png", alt="Custom Tab with five custom menu items.", width="320", height="693", class="screenshot screenshot--filled" %}
+
+To add a menu item, call [`CustomTabsIntent.Builder#addMenuItem`](https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#addMenuItem(java.lang.String,android.app.PendingIntent)) with title and a [`PendingIntent`](https://developer.android.com/reference/android/app/PendingIntent). When the user taps on a menu item, the browser will fire the `PendingIntent`.
+
+
+```java
+CustomTabsIntent intent = new CustomTabsIntent.Builder()
+        ...
+        .addMenuItem("Menu item 1", pendingIntent)
+        .addMenuItem("Menu item 2", pendingIntent)
+        .addMenuItem("Menu item 3", pendingIntent)
+        .addMenuItem("Menu item 4", pendingIntent)
+        .addMenuItem("Menu item 5", pendingIntent)
+        .build();
+```
+
+## Customize the close button
+
+To better fit a Custom Tab into the flow of your app, customize the close button. If you want the user to feel like Custom Tabs is a modal dialog, use the default `“X”` button. If you want the user to feel the Custom Tab is part of the application flow, use the back arrow.
+
+```java
+CustomTabsIntent.Builder intentBuilder = new CustomTabsIntent.Builder();
+intentBuilder.setCloseButtonIcon(BitmapFactory.decodeResource(
+    getResources(), R.drawable.ic_arrow_back));
+```
+
+## Add a bottom toolbar
+
+The bottom toolbar is a very flexible way to add more functionality to a custom tab. 
+
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/e0evUtBxkP0uI7kVj8PS.png", alt="Custom Tab with a bottom toolbar", width="320", height="693", class="screenshot screenshot--filled" %}
+
+By passing a [`RemoteViews`](https://developer.android.com/reference/android/widget/RemoteViews?cmdf=android+remoteviews) object to [CustomTabIntent.Builder.setSecondaryToolbarViews()](https://developer.android.com/reference/android/support/customtabs/CustomTabsIntent.Builder.html#setSecondaryToolbarViews(android.widget.RemoteViews,%20int[],%20android.app.PendingIntent)), the bottom toolbar can be fully customized and dynamically updated.
+
+First, declare a toolbar layout by creating a new layout file `res/layout/custom_tab_toolbar.xml`:
```

Comment:
```suggestion
First, declare a toolbar layout by creating a new layout file, `res/layout/custom_tab_toolbar.xml`:
```

Reply:
The comma is needed because the file name is a restatement of the previous noun. The alternative is to replace the comma with the word 'called'. So:

>... by creating a new layout file called `res/layout/custom_tab_toolbar.xml`